finals-2024-01-15/code/5105-angela-cekovska/src/app/modules/client.ts

```diff
@@ -0,0 +1,16 @@
+export interface Movie {
```

Comment:
Nice interfaces, even as there is no difference between the server and client objects. +5 points